src/pages/signup-affiliates-details/common/_validation.js

```diff
@@ -15,17 +14,15 @@ const validation_is_exceed_number = (input, maxDigit) => {
 
 const validation_is_lack_number = (input, minDigit) => input.length + 1 > minDigit
 
-const nameValidation = (input, fieldName, minDigit, maxDigit, isUserName) => {
+const nameValidation = (input, fieldName, minDigit, maxDigit) => {
```

Comment:
fieldName, minDigit, maxDigit should be in snake case instead.

Reply:
Replaced everywhere in `_validation.js`